src/DXRMissions/DeusEx/Classes/DXRMissionsM02.uc

```diff
@@ -286,6 +286,14 @@ function AnyEntry()
     }
 }
 
+function AddMissionGoals()
+{
+    if (dxr.localURL == "02_NYC_BATTERYPARK") {
+        GiveGoalFromConv(player(), 'FindAmbrosia', 'AnnaDock');
+        RemoveGoalFromConv('FindAmbrosia', 'AnnaDock');
```

Comment:
This will only be removed from the goal the first time you enter the map, then it will continue to be present after any reload or travel back to the map.  This needs to be in AnyEntry.